invenio_communities/assets/semantic-ui/js/invenio_communities/community/search.js

```diff
@@ -124,14 +128,33 @@ export const CommunitiesResults = ({
               }}
             />
           </Grid.Column>
-          <Grid.Column textAlign="right" width={4}>
+          <Grid.Column className="mobile only" width={16} textAlign="center">
```

Comment:
maybe it has been discussed already, could it be an idea to add a `common` folder and add it in there so it can be reused? I think they are identical.